### PR TITLE
[1979687] Make postEphemeral require at least one user in params

### PR DESCRIPTION
**Pull request by popovic-milica**

### CODE REVIEW

File: pumble-sdk/src/api/v1/MessagesApiClientV1.ts [hunk 1]
```diff
@@ -54,7 +54,7 @@ export class MessagesApiClientV1 extends BaseApiClient {
     public async postEphemeral(
         channelId: string,
         payload: V1.SendMessagePayload,
-        ...toUsers: string[]
+        ...toUsers: [string, ...string[]]
```
Comment (xhixha-m) [inline pumble-sdk/src/api/v1/MessagesApiClientV1.ts:57]: Wouldn't it be better separating into two params instead ? 
targetUser, otherUsers . or something like that

Comment (popovic-milica) [reply pumble-sdk/src/api/v1/MessagesApiClientV1.ts]: Changed it

Comment (xhixha-m) [reply pumble-sdk/src/api/v1/MessagesApiClientV1.ts]: Please check if there's some example file in the codebase to make sure all usages have been updated

Comment (popovic-milica) [reply pumble-sdk/src/api/v1/MessagesApiClientV1.ts]: Checked it, all usages seem proper

**Review by xhixha-m — APPROVED** [review]

**Pull request merged**